cms/djangoapps/contentstore/views/assets.py

```diff
@@ -81,6 +82,7 @@ def _asset_index(request, course_key):
     course_module = modulestore().get_course(course_key)
 
     return render_to_response('asset_index.html', {
+        'waffle_flag_enabled': NewAssetsPageFlag.feature_enabled(course_key),
```

Comment:
So in python, we just pass the effective value of the waffle flag through to the mako template